src/microbe_stage/gui/PatchDetailsPanel.cs

```diff
@@ -820,6 +843,16 @@ private void MigrationAmountSliderChanged(double value)
         UpdateMigrationStatusText();
     }
 
+    private float CalculateOtherCompoundAmount()
+    {
+        if (SelectedPatch == null)
+            return 0;
+
+        return (float)Math.Floor(10 * (100.0f - (GetCompoundAmount(SelectedPatch, carbondioxideCompound)
```

Comment:
Again, here I'd really like you to write out this in like 10 lines so that it is very easy to add new compounds to be taken into account here.